### PR TITLE
Taxonomy

### PR DESCRIPTION
**Pull request by nastasi-oq**
Add oq-platform-taxonomy app.
There are 2 companions:
  * oq-platform-taxonomy: https://github.com/gem/oq-platform-taxonomy/pull/1
  * oq-platform-taxtweb: https://github.com/gem/oq-platform-taxtweb/pull/16
Tests are green here: https://ci.openquake.org/job/zdevel_oq-platform-standalone/85/

### CODE REVIEW

**Comment by daniviga** [comment]
Please update https://github.com/gem/oq-engine/blob/master/openquake/server/local_settings.py.standalone too (a PR isn't needed)

File: verifier.sh [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+!/bin/bash
```
Comment (daniviga) [inline verifier.sh]: Missing `#`

File: verifier.sh [hunk 2]
```diff
@@ -422,8 +423,9 @@ echo "\$server" > /tmp/server.pid
 cp openquakeplatform/test/config/moon_config.py.tmpl openquakeplatform/test/config/moon_config.py
 export PYTHONPATH=\$(pwd):\$(pwd)/../oq-moon:\$(pwd)/openquakeplatform/test/config
 export DISPLAY=:1
-python -m openquake.moon.nose_runner --failurecatcher dev -v --with-xunit --xunit-file=xunit-platform-dev.xml  openquakeplatform/test
+python -m openquake.moon.nose_runner --failurecatcher dev -v --with-xunit --xunit-file=xunit-platform-dev.xml  openquakeplatform/test # || true
```
Comment (daniviga) [inline verifier.sh:426]: is `|| true` intentionally commented out?

File: verifier.sh [hunk 3]
```diff
@@ -600,18 +603,24 @@ cd oq-platform-taxtweb
...
 # add a simulated qgis uploaded layer
-./openquakeplatform/bin/simqgis-layer-up.sh --sitename "http://oq-platform.localdomain"
+./openquakeplatform/bin/simqgis-layer-up.sh --sitename \"http://oq-platform.localdomain\"
 
 export PYTHONPATH=\$(pwd):\$(pwd)/openquakeplatform/test/config
 sed 's@^pla_basepath *= *\"http://localhost:8000\"@pla_basepath = \"http://oq-platform.localdomain\"@g' openquakeplatform/test/config/moon_config.py.tmpl > openquakeplatform/test/config/moon_config.py
 export DISPLAY=:1
 python -m openquake.moon.nose_runner --failurecatcher prod -v --with-xunit --xunit-file=xunit-platform-prod.xml  openquakeplatform/test
 sleep 3
+sleep 40000 || true
```
Comment (daniviga) [inline verifier.sh]: why `sleep` 40 secs? Isn't too much? And why it has `|| true`? It should always return true

File: verifier.sh (hunk 2 above)
Comment (nastasi-oq) [reply verifier.sh:426]: Intentionally, to be able to continue interactively if tests fails on the CI lxc.

**Review by daniviga — APPROVED** [review]
LGTM

**Pull request merged**